packaging/bundle-validation/ci_run.sh

```diff
@@ -80,7 +80,7 @@ elif [[ ${SPARK_RUNTIME} == 'spark3.5.0' && ${SCALA_PROFILE} == 'scala-2.13' ]];
   HADOOP_VERSION=3.3.5
   HIVE_VERSION=3.1.3
   DERBY_VERSION=10.14.1.0
-  FLINK_VERSION=1.19.0
+  FLINK_VERSION=1.20.0
```

Comment:
The image tag `IMAGE_TAG` needs to be updated to properly test Hudi Flink bundle on the latest Flink version.

Reply:
updated.

Reply:
It looks like the `testHoodieFlinkQuickstart` is failing after the image tag update, will retrigger the tests and see what happens.